app/src/main/java/ru/flicksbox/movie/domain/MovieEntity.kt

```diff
@@ -0,0 +1,30 @@
+package ru.flicksbox.movie.domain
+
+data class MovieEntity(
+    val actors: List<MovieEntityActor>?,
+    val contentID: Int,
+    val countries: List<MovieEntityCountry>?,
+    val description: String,
+    val directors: List<MovieEntityDirector>?,
+    val genres: List<MovieEntityGenre>?,
+    val id: Int,
+    val images: String,
+    val isFavorite: Boolean,
+    val isLike: Boolean,
+    val isFree: Boolean,
+    val name: String,
+    val originalName: String,
+    val rating: String,
+    val shortDescription: String,
+    val type: String,
+    val video: String,
+    val year: Int,
+)
+
+data class MovieEntityActor(val id: Int, val name: String)
+
+data class MovieEntityCountry(val id: Int, val name: String)
+
+data class MovieEntityDirector(val id: Int, val name: String)
+
+data class MovieEntityGenre(val id: Int, val name: String)
```

Comment:
I suggest to keep `Entity` as postfix, it's much more readable

`MovieActorEntity`
`MovieCountryEntity`
etc.